src/App.js

```diff
@@ -1,13 +1,15 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
+import AstroContainer from './components/Astronomy/AstroContainer';
+import AsteroidContainer from './components/Asteroids/AsteroidContainer';
+import axios from 'axios';
```

Comment:
There's no need to import this where you aren't using it. It is potentially displaying a yellow message in your console bc of it